ChairManager/Data/HashesOutput.xml

```diff
@@ -0,0 +1,36 @@
+<files>
+    <file path="LevelOutput\Campaign\EndGame\level.pak"
+          hash="532439abc6f2dfd937c4ee2ec4f014f0da6d21d26c74af853c9b47ec63f91a3a" type="level"/>
+    <file path="LevelOutput\Campaign\Engineering\CargoBay\level.pak"
```

Comment:
Is this a generated file? Hashes will be different on different versions of the game.

Reply:
these are the hashes that are for a natural state for the chairloader provided PreyFiles. After 1 merge they will fall back in line with the provided files. May need to be rethought when/if we switch to preditor-unpack

Reply:
It's fine for now. The hashes can later be generated by preditor